gaphor/diagram/profiles/tests/test_extensionconnect.py

```diff
@@ -1,63 +1,78 @@
-"""
-Extension item connection adapter tests.
-"""
+"""Extension item connection adapter tests."""
 
 from gaphor import UML
 from gaphor.diagram.classes.klass import ClassItem
 from gaphor.diagram.profiles.extension import ExtensionItem
-from gaphor.tests import TestCase
+from gaphor.diagram.tests.fixtures import allow, connect
 
 
-class ExtensionConnectorTestCase(TestCase):
-    """
-    Extension item connection adapter tests.
-    """
+def test_glue(element_factory, diagram):
+    """Test extension item glue."""
+
+    # GIVEN an ExtensionItem, Stereotype, and Class
```

Comment:
I don't think the GWT comments work for me. They basically state what the next lines of code do.

I tend to separate the G/W/T parts by empty lines. If the code is not clear on this point, I like to create small functions with a descriptive name instead.

Reply:
@amolenaar Since I am less familiar with the code base, I find they help me a ton. However, I understand that they can also get a bit excessive, so I can pull them back out.

Reply:
If there's any way we can improve the tests, make them more readable, we should do that. Comments are a last resort. What can we do to make the code more explanatory?

Reply:
Understood, I am good only using comments as only a last resort :+1: